src/Core/src/Platform/iOS/PickerExtensions.cs

```diff
@@ -61,6 +62,11 @@ internal static void UpdatePicker(this MauiPicker platformPicker, IPicker picker
 			}
 
 			pickerView?.Select(Math.Max(selectedIndex, 0), 0, true);
+
+			if (shouldInvalidateMessure)
+			{
+				picker.InvalidateMeasure();
+			}
```

Comment:
Messure -> Measure (typo)
But since it is initialized 20 lines above its first use, it is even easier to understand and maintain when simply inlining the condition

```suggestion
			if (newSelectedIndex != picker.SelectedIndex)
			{
				picker.InvalidateMeasure();
			}
```

Reply:
This cannot be the case because on the line 57 we assign selectedIndex (newSelectedIndex if not null)